ttnn/cpp/ttnn/operations/eltwise/unary/unary_pybind.hpp

```diff
@@ -1450,19 +1446,43 @@ void py_module(py::module& module) {
         ttnn::hardswish,
         "scale", "Scale value", 1.0f/6.0f,
         "shift", "Shift value", 0.5f,
-        R"doc(Performs hardswish function on :attr:`input_tensor`, :attr:`scale`, :attr:`shift`.)doc");
+        R"doc(
+        Note:
+            Supported dtypes, layouts, and ranks:
+
+            +----------------------------+---------------------------------+-------------------+
+            |     Dtypes                 |         Layouts                 |     Ranks         |
+            +----------------------------+---------------------------------+-------------------+
+            |    BFLOAT16                |          TILE                   |      2, 3, 4      |
+            +----------------------------+---------------------------------+-------------------+)doc");
     detail::bind_unary_composite_floats_with_default(
         module,
         ttnn::hardsigmoid,
         "scale", "Scale value", 1.0f/6.0f,
         "shift", "Shift value", 0.5f,
-        R"doc(Performs hardsigmoid function on :attr:`input_tensor`, :attr:`scale`, :attr:`shift`.)doc");
+        R"doc(
+        Note:
+            Supported dtypes, layouts, and ranks:
+
+            +----------------------------+---------------------------------+-------------------+
+            |     Dtypes                 |         Layouts                 |     Ranks         |
+            +----------------------------+---------------------------------+-------------------+
+            |    BFLOAT16                |          TILE                   |      2, 3, 4      |
+            +----------------------------+---------------------------------+-------------------+)doc");
     detail::bind_unary_composite_floats_with_default(
         module,
         ttnn::hardtanh,
         "min", "min value", -1.0f,
         "max", "max value", 1.0f,
-        R"doc(Performs hardtanh function on :attr:`input_tensor`, :attr:`min`, :attr:`max`.)doc");
+        R"doc(
```

Comment:
Just send the note information string alone, remove `Note`

Reply:
If I add note in the information string, I could not get the table inside the blue box as @yan-zaretskiy suggested. That's why I kept it there.